dev/.rat-excludes

```diff
@@ -1,3 +1,4 @@
+.github/*
```

Comment:
It's very strange, how `.github/*` worked and `.github/pull_request_template.md` didn't 🤔